app/src/components/AnalysisPane.js

```diff
@@ -1,147 +1,194 @@
+        '&:hover svg': {
+          color: '#fff',
+        },
+      }}
+    />
+  );
 
   const attrs = useMemo(() => {
-    const a = [ [ 'Type', displayNames[result.Type] ] ];
+    const a = [];
 
-    (result.Attributes || []).forEach(attr => {
-      a.push([
-        displayNames[attr.Type],
-        attr.Text
-      ]);
+    (result.Attributes || []).forEach((attr) => {
+      a.push([displayNames[attr.Type], attr.Text]);
     });
-
-    if (result.ICD10CMConcepts) {
-      a.push([
-        'ICD-10-CM Concepts',
-        <ConceptTable rows={result.ICD10CMConcepts} />
-      ]);
-      // Code, Description, Score
-    }
-
-    if (result.RxNormConcepts) {
-      // Code, Descroption, Score
-      a.push([
-        'RxNorm Concepts',
-        <ConceptTable rows={result.RxNormConcepts} />
-      ]);
-    }
-
     return a;
-  }, [ result ]);
+  }, [result]);
+
+  if (!result.ICD10CMConcepts && !result.RxNormConcepts && !result.Attributes) {
+    return (
+      <Flex width='100%' alignItems='center'>
+        <Flex
+          flex='1'
+          mr={2}
+          height='2.5rem'
+          border={result.Score && result.Score < 0.5 ? '1px solid red' : '1px solid grey'}
+          bg='white'
+          px={4}
+          alignItems='center'
+        >
+          {result.Text} {result.Type ? '|' : null} {displayNames[result.Type]}
+        </Flex>
+        {closeIcon}
+      </Flex>
+    );
+  }
+
+  if (!result.ICD10CMConcepts && !result.RxNormConcepts && result.Attributes) {
+    return (
+      <Flex width='100%' alignItems='center'>
+        <Flex
+          flex='1'
+          mr={2}
+          height='2.5rem'
+          border={result.Score && result.Score < 0.5 ? '1px solid red' : '1px solid grey'}
+          bg='white'
+          px={4}
+          alignItems='center'
+        >
+          {attrs.map(([key, value]) => (
+            <React.Fragment key={key}>
+              {result.Text} {value ? '|' : null} {value}
+            </React.Fragment>
+          ))}
+        </Flex>
+
+        {closeIcon}
+      </Flex>
+    );
+  }
 
+  const concepts = [...(result.ICD10CMConcepts ? result.ICD10CMConcepts : result.RxNormConcepts)];
+  concepts.sort(function (concept1, concept2) {
+    return concept2.Score - concept1.Score;
+  });
 
   return (
-    <div className={cs(s.result, isExcluded && s.excluded)}>
-      <button className={s.removeButton} onClick={onHide} aria-label="Exclude this item from the export" />
-      <h4>{result.Text}</h4>
-
-      <dl>
-        {attrs.map(([ key, value ]) => (
-          <>
-            <dt>{key}</dt>
-            <dd>{value}</dd>
-          </>
+    <Flex width='100%' alignItems='center'>
+      <Select
+        mr={2}
+        border='1px solid'
+        borderColor={concepts[0].Score < 0.5 ? 'red' : 'grey'}
+        borderRadius='0'
+        bg='white'
+        _hover={{ borderColor: concepts[0].Score < 0.5 ? 'red' : 'grey', boxShadow: 'none' }}
+      >
+        {concepts.map((concept) => (
```

Comment:
Destructuring the `(concept) =>` to `({ Code, Text, Description }) => ` should make this a bit cleaner